src/blockchain/bitcoin/p2p/Bitcoin.cpp

```diff
@@ -67,21 +67,33 @@ const CommandMap command_map_{
 };
 const CommandReverseMap command_reverse_map_{reverse_map(command_map_)};
 
-const OTData AddressVersion::cjdns_prefix_{[]() {
-    auto out = Data::Factory();
-    out->DecodeHex("0xfc");
+auto AddressVersion::cjdns_prefix() -> OTData
+{
+    static auto out = []() {
```

Comment:
I know that the names of the variables are obscured here but it makes I read that code three times instead one

Reply:
Im open for suggestions:D